### PR TITLE
Feat: Add media query breakpoints mixin

### PR DESCRIPTION
**Pull request by yxwangOCR**
#### Ticket number [#1] : https://trello.com/c/7svvHbfH
- [x] I have performed a self-review of my code

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -6,6 +6,18 @@ module.exports = {
...
+    },
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:storybook/recommended",
+  ],
+  ignorePatterns: ["./storybook", "vite.*.ts", "vite-env.d.ts", "./vite"],
+=======
```
Comment (reboottime) [inline .eslintrc.cjs:20]: Conflicts files are submitted

**Review by reboottime — CHANGES_REQUESTED** [review]

File: src/sass/abstract/_breakpoints.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@use "sass:map";
+
 $grid-breakpoints: (
   xs: 0,
```
Comment (ixtk) [inline src/sass/abstract/_breakpoints.scss:4]: I don't see the benefits of using 0 as the value here. Something like 320px would make more sense as an extra small value.

**Review by ixtk — CHANGES_REQUESTED** [review]

File: src/sass/abstract/_breakpoints.scss (hunk 2 above)
Comment (reboottime) [reply src/sass/abstract/_breakpoints.scss:4]: good call, thank you very much

File: .eslintcache [hunk 3]
```diff
@@ -0,0 +1 @@
...27,"suppressions":"139"},{"ruleId":"140","severity":2,"message":"141","line":10,"column":11,"nodeType":"142","messageId":"143","endLine":10,"endColumn":16,"suppressions":"144"},{"ruleId":"140","severity":2,"message":"141","line":10,"column":11,"nodeType":"142","messageId":"143","endLine":10,"endColumn":16,"suppressions":"145"},{"ruleId":"146","severity":2,"message":"147","line":52,"column":9,"nodeType":"148","messageId":"137","endLine":52,"endColumn":21,"suppressions":"149"},{"ruleId":"146","severity":2,"message":"147","line":63,"column":9,"nodeType":"148","messageId":"137","endLine":63,"endColumn":21,"suppressions":"150"},"@typescript-eslint/no-empty-function","Unexpected empty arrow function.","ArrowFunctionExpression","unexpected",["151"],["152"],"@typescript-eslint/no-empty-interface","An empty interface is equivalent to `{}`.","Identifier","noEmpty",["153"],["154"],"no-console","Unexpected console statement.","MemberExpression",["155"],["156"],{"kind":"157","justification":"158"},{"kind":"157","justification":"158"},{"kind":"157","justification":"158"},{"kind":"157","justification":"158"},{"kind":"157","justification":"158"},{"kind":"157","justification":"158"},"directive",""]
```
Comment (reboottime) [inline .eslintcache:1]: Can you please ignore this file in .gitignore?

here your personal information is leaked

image

File: jest.setup.js [hunk 4]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```
Comment (reboottime) [inline jest.setup.js:1]: conflict changes are submitted, can you resolve the conflicts?

**Review by reboottime — CHANGES_REQUESTED** [review]

File: src/sass/abstract/_breakpoints.scss [hunk 5]
```diff
@@ -6,3 +8,39 @@ $grid-breakpoints: (
...
+}
+
+@mixin xl {
+  @media screen and (min-width: map.get($grid-breakpoints, "xl")) {
+    @content;
+  }
+}
+
+@mixin xlg {
+  @media screen and (min-width: map.get($grid-breakpoints, "xl")) {
```
[the remark this answers is not in the document]
Comment (yxwangOCR) [reply src/sass/abstract/_breakpoints.scss]: Fixed.

File: src/sass/abstract/_breakpoints.scss (hunk 2 above)
Comment (yxwangOCR) [reply src/sass/abstract/_breakpoints.scss]: Fixed.

**Review by reboottime — APPROVED** [review]

**Comment by reboottime** [comment]
Approved and merged to sass branch as I need to resolve some issues mannually

**Pull request merged**